src/base/bittorrent/torrenthandle.h

```diff
@@ -447,12 +447,15 @@ namespace BitTorrent
 
         enum StartupState
         {
-            NotStarted,
+            Initialized,
+            PreparingPaused,
```

Comment:
And also explain the state name meanings in comments.

Reply:
I put more or less detailed description.